src/complete.js

```diff
@@ -0,0 +1,9 @@
+export function checked(id, arr) {
+  arr[id].completed = true;
```

Comment:
[Optional] The boolean "completed" could be change with the negation operator ! like this arr[id].completed = !arr[id].completed so it will comply with the DRY principle avoiding repeating the same code but reverse for the unchecked case.